metabase/metabase
# Change `Export as PDF` button name for dashboards with multiple tabs

Resolves https://github.com/metabase/metabase/issues/31228

### Description

Currently, we have a button in the dashboard options called `Export as PDF`, but only exports the current tab rather than the whole dashboard. Because of the implementation of our PDF exports, we're not at a point where we can export the whole dashboard in one go. Instead, for now, we'll change the button name to explicitly specify the tab when the user has multiple tabs in their dashboard.

Also added tests in `DashboardHeader.unit.spec.jsx`


### How to verify


To see that a dashboard with one tab shows `Export as PDF`
1. Create a dashboard
2. Save the dashboard
3. Click the `...` icon on the top right
4. You should see `Export as PDF`

Now, to see that the button name changes when we have multiple tabs in the dashboard:
1. Edit the dashboard
2. Add a tab to the dashboard by clicking the `+` button in the tab header
3. Save the dashboard
4. Click the `...` icon on the top right
5. You should now see the button say `Export tab as PDF`


### Demo

https://github.com/metabase/metabase/assets/25306947/6d1bfe4a-909a-4b19-92e6-36d48d098640


### Checklist
